services/searchengine/bleveengine/common.go

```diff
@@ -125,6 +125,13 @@ func BLVPostFromPostForIndexing(post *model.PostForIndexing) *BLVPost {
 	}
 }
 
+func splitFilenameWords(name string) string {
```

Comment:
Given that this function is already in the enterprise code too we can move it to utils and use it in both places

Reply:
I prefer to keep some duplication there than moving that to `utils` which means nothing, and is going to add an artificial (and unneeded) coupling with the `utils` package. I'm ok moving this into the searchengine package, but I don't see the need.